service/biz/tpm_cert_verifier_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package biz
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestVerifyAndParseIakAndIDevIdCerts(t *testing.T) {
+	/*
+		TODO(jenia-grunin):
+		* Generate certs/keys on the fly.
+		* Tests to add:
+			- Unsupported algo such as DSA.
+			- Bad pub key length for RSA.
+			- Bad pub key length for ECC.
+			- IAK and IDevID cert serials don't match.
+			- Bad cert pem header.
+			- Good pem cert header, but bad x509 structure.
+			- Repeat above for both IAK and IDevID certs.
+	*/
+	iakCertPemRsa2048 := `-----BEGIN CERTIFICATE-----
```

Comment:
I will merge this for now, and in the next PR I will add utilities to generate certs on the fly according to provided options. This will be useful for all those tests that I left out in the TODO comment :)